### PR TITLE
Add glooctl get ratelimitconfigs

### PR DESCRIPTION
**Pull request by ashleywang1**
# Description

Add the `glooctl get ratelimitconfigs` command.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/4085

### CODE REVIEW

File: projects/gloo/cli/pkg/constants/commands.go [hunk 1]
```diff
@@ -50,6 +50,11 @@ var (
 		Aliases: []string{"ac", "authconfig"},
```
Comment (arianaw66) [inline projects/gloo/cli/pkg/constants/commands.go:50]: While you're at it, should this one and the `secret` alias be plural?

**Review by arianaw66 — COMMENTED** [review]
I'm a little confused by the issue title- is this supposed to also add more docs for the RateLimitConfig type? Otherwise lgtm.

**Comment by arianaw66** [comment]
Wait, doesn't this need to print actions/setActions too?

**Comment by ashleywang1** [comment]
> I'm a little confused by the issue title- is this supposed to also add more docs for the RateLimitConfig type? Otherwise lgtm.

There is a step to generate docs for Gloo CLI commands! Those were the docs I was referring to.

**Comment by ashleywang1** [comment]
Currently, the output looks like this:
Screen Shot 2021-01-13 at 2 33 29 PM

I'm not too clear on what information in rate limit configs is the most useful, so suggestions are appreciated!

File: projects/gloo/cli/pkg/printers/ratelimitconfig.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		}
+	}
+
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.Render()
+}
+
+func SetRateLimitConfig(list ratelimit.RateLimitConfigList, w io.Writer) {
+	table := tablewriter.NewWriter(w)
+	table.SetHeader([]string{"Set-style RateLimitConfig", "SetDescriptors", "SetActions"})
```
Comment (arianaw66) [inline projects/gloo/cli/pkg/printers/ratelimitconfig.go:65]:
```suggestion
	table.SetHeader([]string{"Set-style RateLimitConfig", "Set Descriptors", "Set Actions"})
```

File: projects/gloo/cli/pkg/printers/ratelimitconfig.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+	for _, ratelimitConfig := range list {
+		name := ratelimitConfig.GetMetadata().Name
+		actions := printActions(ratelimitConfig.Spec.GetRaw().GetRateLimits())
+		maxNumLines := len(actions)
+		descriptors := printDescriptors(ratelimitConfig.Spec.GetRaw().GetDescriptors())
+		if len(descriptors) > maxNumLines {
+			maxNumLines = len(descriptors)
+		}
+
+		// If
```
Comment (arianaw66) [inline projects/gloo/cli/pkg/printers/ratelimitconfig.go:39]:
```suggestion
```

**Comment by arianaw66** [comment]
There doesn't appear to be a distinction between 2 descriptors on the same level that's not the top level vs a descriptor and another nested under it, but I think that's due to the .String() call which uses an external library, so probably not worth fixing?

**Review by kdorosh — APPROVED** [review]

**Pull request merged**